### PR TITLE
Update pt-PT.json

### PR DESCRIPTION
**Pull request by smarquespt**

### CODE REVIEW

File: localization/languages/pt-PT.json [hunk 1]
```diff
@@ -104,22 +104,22 @@
...
-        "settingsContentBlockingExceptions": null, //missing translation
+        "settingsContentBlockingLevel0": "Permitir todos os anúncios e 'trackers'",
+        "settingsContentBlockingLevel1": "Bloquear anúncios e trackers de terceiros",
+        "settingsContentBlockingLevel2": "Bloquear todos os anúncios e 'trackers'",
+        "settingsContentBlockingExceptions": "Permitir anúncios nestes sites:",
         "settingsBlockScriptsToggle": "Bloquear scripts",
         "settingsBlockImagesToggle": "Bloquear imagens",
         "settingsEasyListCredit": {
-            "unsafeHTML": null //missing translation
+            "unsafeHTML": "Baseado em EasyList e EasyPrivacy, criadas por <a target=\"_blank\" href=\"https://easylist.to/\">The EasyList authors</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>)."
```
Comment (PalmerAL) [inline localization/languages/pt-PT.json]: Could you translate the "view license" part here as well?

Comment (smarquespt) [reply localization/languages/pt-PT.json]: I can, please merge this one. I'll make another one after.

Comment (PalmerAL) [reply localization/languages/pt-PT.json]: OK, sure.

**Comment by smarquespt** [comment]
Done in https://github.com/minbrowser/min/pull/731

**Pull request merged**